CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v3.23.0
+
+### Added
+
+- Source `pledge.fnp` [#117]
+
```

Comment:
Add entries about fields, that are fillable by new source, pls
Like 
```
### Changed

- Field with path `%field_path%` also fillable by `pledge.fnp` source [#117]
```

Reply:
Added